intutil/math.go

```diff
@@ -103,3 +103,12 @@ func (decrementor *Decrementor) GetInitialMax() int {
 func Clamp[T constraints.Ordered](value, min, max T) T {
 	return Min(Max(value, min), max)
 }
+
+// Anonymize converts an array of ints to an array of anonymous interfaces
+func Anonymize(arr []int) []interface{} {
```

Comment:
Let's make one that works for all types and merge this and the stringutil one.
```suggestion
func Anonymize[T any](arr []T) []interface{} {
```

Reply:
@apynes what package should that go in? Seems like it could go in the database package since we are really only using it for In statements, could maybe shoehorn into the collection package, or it could go in a new package called `sliceutil` or something

Reply:
`sliceutil` or `generics` or something `sliceutil` is probably better. @morus12 what do you think?